apps/src/applab/applab.js

```diff
@@ -1121,6 +1121,10 @@ Applab.execute = function() {
       var session = studioApp.editor.aceEditor.getSession();
       annotationList.attachToSession(session, studioApp.editor);
       annotationList.clearRuntimeAnnotations();
+      studioApp.editor.aceEditor.session.on("change", function () {
```

Comment:
I believe this covered both cases (at least I have vague memories of testing that) but can validate for sure.